packages/database/src/migrations/20210211041322-AddSubFacilityEntityType-modifies-schema.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+import { TupaiaDatabase } from '../TupaiaDatabase';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = async function () {
+  const db = new TupaiaDatabase();
+  await db.executeSql(`
+    ALTER TYPE public.entity_type ADD VALUE IF NOT EXISTS 'sub_facility';
+  `);
+  db.closeConnections();
```

Comment:
`db.closeConnections();` should we be doing this at the end of migrations? If so, woops I haven't been doing it, will start hehe

Reply:
This migration was actually just a copy paste from other migrations that adds new entity type (eg: https://github.com/beyondessential/tupaia/pull/568 or https://github.com/beyondessential/tupaia/pull/368). I'm also curious but I'm sure there was a reason for it so I didn't bother changing it 😄